spec/game_runner_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'spec_helper.rb'
+require_relative '../lib/game_runner.rb'
+
+RSpec.describe Game, "#run" do
+  def grid_format
```

Comment:
This happened sometime during my review. It makes the cognitive overhead a bit easier. But there are much easier ways of making it really clear what the output looks like.

Reply:
Got it. What do you think of what's below for the first iteration?